### PR TITLE
Added missing scaffolder management permission

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added `templateManagementPermission` to `scaffolderTemplatePermissions` array so it will get pickup by the `permissionIntegrationRouter`

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by acierto — APPROVED** [review]

File: plugins/scaffolder-common/src/permissions.ts [hunk 1]
```diff
@@ -156,5 +157,4 @@ export const scaffolderPermissions = [
   ...scaffolderTemplatePermissions,
   ...scaffolderActionPermissions,
   ...scaffolderTaskPermissions,
-  templateManagementPermission,
```
Comment (vinzscam) [inline plugins/scaffolder-common/src/permissions.ts:159]: mmm this used to be correct 🤔 
I think the problem should be somewhere else then

**Review by vinzscam — CHANGES_REQUESTED** [review]

File: plugins/scaffolder-common/src/permissions.ts (hunk 1 above)
Comment (vinzscam) [reply plugins/scaffolder-common/src/permissions.ts:159]: So I think the way to go would be to pass `permissions: scaffolderPermissions` here: https://github.com/backstage/backstage/blob/c34fd4ed4ab9ac2e90d51c988d259b5e4be4f0b6/plugins/scaffolder-backend/src/service/router.ts#L448
but I'm not sure whether the integration router can properly handle duplicates at this point, as `scaffolderPermissions` contains resources permissions as well.
Short term another way would be to pass `permissions: [...scaffolderPermissions, templateManagementPermission]`

Comment (awanlin) [reply plugins/scaffolder-common/src/permissions.ts:159]: I'll give this a try, with the current fix there was no duplicates based on my testing.

Comment (vinzscam) [reply plugins/scaffolder-common/src/permissions.ts:159]: your fix should work. The issue is to group the new permission under `scaffolderTemplatePermissions` as they are resource permissions of type `scaffolder-template`

**Review by vinzscam — APPROVED** [review]
alright I've added the fix, it should be ready to go now 🚀

**Pull request merged**